### ISSUE TITLE
Customer token not found - HPP

### ISSUE THREAD

**Issue by steve-ico3** [klarna/kco_rest_php#57]
Apologies if this is the wrong place for this, wasn't sure if you had a ticket system or anything

**Describe the bug**
I've got as far as the response from the hosted payment page which gives me a token in the url, example: token=55a5605d-7a11-2d2b-a716-a16ec1a6e327

I then use the Klarna\Rest\CustomerToken\Tokens class to attempt to read the token I've received but it just gives me an error every time

TOKEN_NOT_FOUND: Customer token not found (#e11340df-6405-43fa-8110-f8ec6d1086cb)

**Which service do you use**
Hosted Payment Page, Playground environment

**To Reproduce**
Basically this but with the token 55a5605d-7a11-2d2b-a716-a16ec1a6e327 (and my merchant/secret)
https://github.com/klarna/kco_rest_php/blob/v4.x/docs/examples/CustomerTokenAPI/Tokens/read_token_details.php

**Expected behavior**
Fetch the token details so I can check that it's a valid token and go on to create the order

**Additional context**
Playground environment, tried various tokens not just 1

Following this: https://developers.klarna.com/documentation/hpp/guides/kp-integration/

So stuck on step 5 due to the token not found error

**Comment by steve-ico3** [comment]
Can anyone tell me if the CustomerTokenApi is what I should be using at least? 
Keen to get Klarna implemented but can't continue until I can get past this hurdle

**Comment by freethan** [comment]
Hi @steve-ico3, apologies for the belated reply. The token received is an `authorization` token and **not** a `CustomerToken`, hence it is not found when you query for it through the CustomerToken API.

This token allows you to create an order manually through the Klarna Payments API (please see here for the exact call: https://developers.klarna.com/api/#payments-api__create-a-new-order__authorization_token)

In the guide you're following, you can find details on that on steps 4.b + 5, just above https://developers.klarna.com/documentation/hpp/guides/kp-integration/#success-flow

Hope this helps.
Regards

**Comment by steve-ico3** [comment]
Thanks for the reply @freethan that explains a lot!

I've noticed this code is now marked as deprecated on packagist so I'm guessing it's not worth me continuing to use it anyway?

**Comment by freethan** [comment]
Hi @steve-ico3 yes the PHP as well as Java and .NET REST SDKs are now deprecated. Please see https://github.com/klarna/kco_rest_php#workarounds for workarounds.
